js/libs/interface.js

```diff
@@ -251,6 +251,22 @@ const app = new Vue({
         return
       }
 
+    let emptyNameIndex = _.findIndex(this.maps, function(map) {
+      return map.name === ''
+    });
```

Comment:
No semicolons please

```suggestion
    })
```